### ISSUE TITLE
The library should be a library

### ISSUE THREAD

**Issue by acid0078** [s1na/sc2map#17]
We should think about how we could make the library ready for deployment for other projects.

**Comment by s1na** [comment]
We asked Niklas once in the beginning and he said making it a library is not a priority, did he mention this recently?

**Comment by omidn** [comment]
yeah he did. As far as I can remember, he said we don't need to necessarily make a library, But we might think about what is possible in the next meetings.
